srcpkgs/fwupd-efi/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'fwupd-efi'
+pkgname=fwupd-efi
+version=1.1
+revision=1
+build_style=meson
+build_helper="gir"
```

Comment:
I think you should restrict this for the same archs as it used to be in `fwupd`, as well.

Reply:
Sure.